### PR TITLE
Add pandas-profiling recipe

### PR DESCRIPTION
**Pull request by proinsias**
Due to a Windows/Python 3 [issue](aerkalov/ebooklib#104) with the ebooklib dependency, skip the Windows/Python 3 builds for now.

### CODE REVIEW

File: recipes/pandas-profiling/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - matplotlib >=1.4
+
+test:
+  imports:
+    - pandas_profiling
+
+about:
+  home: http://github.com/jospolfliet/pandas-profiling
+  license: MIT License
+  # license_file: LICENSE # When added to MANIFEST.in – https://github.com/JosPolfliet/pandas-profiling/pull/22
```
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: xref: https://github.com/JosPolfliet/pandas-profiling/pull/22

File: recipes/pandas-profiling/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas >=0.16
+    - matplotlib >=1.4
```
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: Are `pandas` and `matplotlib` needed during the build? Or can they be dropped?

File: recipes/pandas-profiling/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
```
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: Why skip Python 3.x on Windows?

File: recipes/pandas-profiling/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    - pandas >=0.16
+    - matplotlib >=1.4
+
+test:
+  imports:
+    - pandas_profiling
+
+about:
+  home: http://github.com/jospolfliet/pandas-profiling
+  license: MIT License
```
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: Please drop `License`.

File: recipes/pandas-profiling/meta.yaml (hunk 3 above)
Comment (proinsias) [inline recipes/pandas-profiling/meta.yaml]: Due to a Windows/Python 3 [issue](aerkalov/ebooklib#104) with the ebooklib dependency, I'm skipping the Windows/Python 3 builds for now.

Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: That link got messed up. Are you referring to issue ( https://github.com/aerkalov/ebooklib/issues/104 )?

File: recipes/pandas-profiling/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    - matplotlib >=1.4
+
+test:
+  imports:
+    - pandas_profiling
+
+about:
+  home: http://github.com/jospolfliet/pandas-profiling
+  license: MIT
+  # license_file: LICENSE # When added to MANIFEST.in – JosPolfliet/pandas-profiling#22
```
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: xref: https://github.com/JosPolfliet/pandas-profiling/pull/22

**Comment by jakirkham** [comment]
Thanks @proinsias.

Had a clarifying question, but it doesn't block merging.

File: recipes/pandas-profiling/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/pandas-profiling/meta.yaml]: So I don't think this uses `ebooklib`, right? Just `pandas` and `matplotlib`.

Please see PR ( https://github.com/conda-forge/pandas-profiling-feedstock/pull/1 ) for enabling Windows builds.

**Pull request merged**